boards/omote/board.c

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2020 Oppila Microsystems - http://www.oppila.in
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     boards_omote
+ * @{
+ *
+ * @file
+ * @brief       Board specific implementations for the omote board
+ * @}
+ */
+
+#include "board.h"
+#include "cpu.h"
+
+static inline void leds_init(void);
+
+void board_init(void)
+{
+    /* initialize the boards LEDs */
+    leds_init();
+    /* initialize the CPU */
+    cpu_init();
+    /* The boot pin must be set to input otherwise it may lock the bootloader */
+    gpio_init(BOOT_PIN, GPIO_IN);
+
+    gpio_init(BTN0_PIN, BTN0_MODE);
```

Comment:
An application using the button will take care of that (and might set up a callback) - you can drop this.

Reply:
This is still unaddressed.

Reply:
@Oppila-ms please address this one too.